src/System.Private.CoreLib/shared/System/Security/Principal/IIdentity.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
- `Name` should return `string?`
- `AuthenticationType` should return `string?`